staging/src/k8s.io/client-go/kubernetes/typed/authentication/v1/fake/fake_tokenreview_expansion.go

```diff
@@ -17,11 +17,17 @@ limitations under the License.
 package fake
 
 import (
+	"context"
+
 	authenticationapi "k8s.io/api/authentication/v1"
 	core "k8s.io/client-go/testing"
 )
 
 func (c *FakeTokenReviews) Create(tokenReview *authenticationapi.TokenReview) (result *authenticationapi.TokenReview, err error) {
+	return c.CreateContext(context.Background(), tokenReview)
+}
+
+func (c *FakeTokenReviews) CreateContext(ctx context.Context, tokenReview *authenticationapi.TokenReview) (result *authenticationapi.TokenReview, err error) {
```

Comment:
Fun.  You're not prepared to update a generator make the API consistent?

Reply:
I still want the generator to produce methods taking a context as well (e.g. https://github.com/kubernetes/community/pull/1166), but these particular Create methods were already hand-written.

Reply:
I was talking with clayton about the shape we want for the generated changes, but I expect that to take quite a bit longer, and I don't want to block this on the generator bits. These clients are the only ones the API server uses internally to call to remote endpoints, so I think it's worth fixing them now.